data/src/main/java/org/apache/iceberg/data/BaseDeleteLoader.java

```diff
@@ -259,4 +284,46 @@ private long estimateEqDeletesSize(DeleteFile deleteFile, Schema projection) {
   private int estimateRecordSize(Schema schema) {
     return schema.columns().stream().mapToInt(TypeUtil::estimateSize).sum();
   }
+
+  private boolean containsDVs(Iterable<DeleteFile> deleteFiles) {
```

Comment:
Maybe this method should be named `containsSingleDV()` ?

Reply:
Updated.